data-plane/pom.xml

```diff
@@ -73,14 +74,45 @@
 
   <dependencyManagement>
     <dependencies>
+      <!-- Vertx -->
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
         <version>${vertx.version}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-micrometer-metrics</artifactId>
+        <version>${vertx.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.hdrhistogram</groupId>
+            <artifactId>HdrHistogram</artifactId>
+          </exclusion>
+        </exclusions>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+
+      <!-- Micrometer -->
+      <dependency>
+        <groupId>io.micrometer</groupId>
+        <artifactId>micrometer-core</artifactId>
+        <version>${micrometer.version}</version>
+        <type>pom</type>
```

Comment:
You don't need the type pom scope import, that' only for boms

Reply:
More than manually importing the core and the registry prometheus, why don't you import the bom? https://mvnrepository.com/artifact/io.micrometer/micrometer-bom/1.5.5